### PR TITLE
feat: beautiful icons by @akrych

### PR DESCRIPTION
**Pull request by hacdias**
That's it.

/cc @olizilla

### CODE REVIEW

File: icons/glyph_attention.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><defs><style>.cls-1{fill:#3c3c3b;}</style></defs><title>glyph_attention</title><g id="glyph_attention"><path class="cls-1" d="M99.05,88.91,71.27,40.8a5.84,5.84,0,0,0-10.12,0L33.37,88.91a5.85,5.85,0,0,0,5.06,8.77H94A5.85,5.85,0,0,0,99.05,88.91ZM69,88.26H63.47V82.77H69ZM69,78H63.47V56.24H69Z"/></g></svg>
```
Comment (olizilla) [inline icons/glyph_attention.svg]: If we remove `<style>.cls-1{fill:#3c3c3b;}</style>` and the `class="cls-1" ` then it's simpler to set the color from outside the svg...

```css
.fill-current {
  fill: currentColor
}
```

We can shave a few bytes and remove the id where they aren't being reference as well

**Comment by hacdias** [comment]
@olizilla just made some changes. The class `fill-color` should be added to the `<svg>` element.

**Comment by hacdias** [comment]
Although I'm more into creating a font with the icons and then having a class for each icon.

**Comment by hacdias** [comment]
Can we merge this?

**Comment by hacdias** [comment]
Ping @olizilla :D

**Comment by olizilla** [comment]
@hacdias this looks good. Can you resolve the conflicts and remove the transition from the hovers and it's good to go.

**Comment by hacdias** [comment]
@olizilla rebased!! 😄

**Comment by olizilla** [comment]
ok, i'll merge this and remove the transition as that can be done with a separate atom.

**Pull request merged**